nixos/modules/services/mail/protonmail-bridge.nix

```diff
@@ -0,0 +1,45 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+with lib;
```

Comment:
```suggestion
```
It's best to avoid such wide `with` scoping. Please call `lib` directly whenever you need it.
https://nix.dev/guides/best-practices.html#with-scopes

Reply:
fixed